### DISCUSSION TITLE
Issues with importing modules

### DISCUSSION THREAD

**Discussion by janober** [pyodide/pyodide#3551] [General]
Hello,

Sadly another issue. Often when I try to import modules in pyodide (running in Node.js 16.13) I get very long error message which amount to:
`name '_jsfetch' is not defined`
Do you know why do I get them and how can I fix those imports?

Thanks a lot!

**Comment by hoodmane** [comment]
Can you provide more context for the error message?

**Reply by hoodmane** [reply]
Nevermind, the problem is that you need to provide a fetch implementation. E.g.,:

```js
if(!globalThis.fetch) {
   globalThis.fetch = require("node-fetch");
}
```

**Reply by janober** [reply]
Great, thank you very much! Will give that a try.
